lib/dynosaur/inputs/abstract_input_plugin.rb

```diff
@@ -64,10 +69,8 @@ def min_recent_values
         return @recent.min
       end
 
-      def get_status
+      def health
         now = Time.now
-        value = get_value
-        estimate = estimated_resources  # minor race condition, but only matters for logging
         health = "OK"
```

Comment:
Using the same name for the variable and the method sounds like some potential future headache